src/components/CenterArea/MainContents.vue

```diff
@@ -2,7 +2,19 @@
 
 <template>
   <div>
-    <!-- サイドメニューで選択したコンテンツの表示 -->
-    <RouterView></RouterView>
+    <transition name="fade" mode="out-in">
+      <!-- サイドメニューで選択したコンテンツの表示 -->
+      <RouterView></RouterView>
```

Comment:
```suggestion
      <RouterView />
```
[nits]こちらのほうがシンプルに見えるかと思います。

Reply:
修正致しました。